tasks/queue-workflow/index.js

```diff
@@ -13,20 +13,22 @@ const { buildExecutionArn } = require('@cumulus/message/Executions');
 *   that is passed to the next task in the workflow
 **/
 async function queueWorkflow(event) {
-  const workflow = event.input.workflow || {};
-  const workflowInput = event.input.workflowInput || {};
+  const workflow = event.config.workflow || {};
```

Comment:
Yeah so this was me figuring out how tasks work. The config rather than the input should really be the task specific setup stuff, so I had to make that adjustment